### PR TITLE
Attitude controllers

### PR DESCRIPTION
**Pull request by nhz2**
Work in progress sensor models and attitude controller. Also I changed main, so it doesn't make plots any more, instead, to plot the trajectory, run main, then run plot_trajectory(truth_trajectory).

### CODE REVIEW

File: MATLAB/main.m
Comment (kylekrol) [inline MATLAB/main.m]: Looks great

**Review by kylekrol — COMMENTED** [review]
I think the overall structure looks pretty good. Not gonna be too terribly picky given so many things are going to be changing so quickly. I'm good with it if @stewartaslan is as well.

Won't approve it for now, but if no one else does and you want to keep moving @nhz2, lmk.

**Comment by stewartaslan** [comment]
Before we merge this PR I'd like to improve performance a little bit, since it no longer runs in real time. Once we improve the performance to something like 2x or 10x real time let's merge this stuff.

**Review by stewartaslan — APPROVED** [review]
Sim now runs at 10-20x real time, since we replaced the call to planetEphemeris with the simple Keplerian model of the Earth's orbit around the sun. The unit test demonstrates that this simplification is always within 0.1 degrees of the truth.

**Pull request merged**
